README.md

```diff
@@ -407,6 +407,18 @@ The `@StateObject` documentation reads:
 
 `@Instantiated`, `@Forwarded`, or `@Received` objects may be decorated with [`@ObservedObject`](https://developer.apple.com/documentation/swiftui/ObservedObject). Note that `@Instantiated` objects declared on a `View` will be re-initialized when the view is re-initialized. You can find a deep dive on SwiftUI view lifecycles [here](https://www.donnywals.com/understanding-how-and-when-swiftui-decides-to-redraw-views/).
 
+#### Dependency cycles
+
+In a manual DI system it is easy to instantiate an object that can instantiate another object that can then instantiate the first object again. Instantiable objects in SafeDI, however, are not able to generate this kind of circular dependency. Instantiable objects can achieve this kind of circular dependency manually: one object in the cycle must call other cycle objects‘ memberwise initializers directly, rather than utilizing generated `Instantiator`s or `ForwardingInstantiator`s.
```

Comment:
imo dependency cycles should be avoided... but it's important to call out what is required to enable their creation for teams that need this.

Reply:
I think I have a way to fix this. But it's going to take time I don't have at the moment. In the interim, we should document this shortcoming.